src/api/manager/inventory.ts

```diff
@@ -20,3 +21,21 @@ export async function getAudioMapping(id: ObjectId): Promise<IResponse> {
       throw `Could not find audio mapping for source: ${id.toString()}`;
     })) as IResponse;
 }
+
+export async function putDeleteFlagOnInventorySourceItem(
```

Comment:
`purgeInventorySourceItem` perhaps? considering it no longer sets a flag per say.